### PR TITLE
First drop of release notes.

### PR DESCRIPTION
**Pull request by MMaiero**
Signed-off-by: Maiero <matteo.maiero@eurotech.com>

### CODE REVIEW

File: kura/distrib/RELEASE_NOTES.txt [hunk 1]
```diff
@@ -1,3 +1,275 @@
...
+Know Issues :
+  * The implementation of the CryptoService performs encryption using a
+    password that is hardcoded and published.
+  * Modem: Ublox Lisa U201 may not be able to establish PPP connection when CHAP/PAP authentication is required.
+  * BLE only tested on the Raspberry Pi B with a Broadcom BCM20702 USB dongle and the TI SensorTag. The kernel version was
+    "Linux raspberrypi 3.18.11-v7+ #781 SMP PREEMPT Tue Apr 21 18:07:59 BST 2015 armv7l GNU/Linux".
+    Note that on the kernel "Linux version 4.1.7-v7+ (dc4@dc4-XPS13-9333) (gcc version 4.8.3 20140303 (prerelease)
+    (crosstool-NG linaro-1.13.1+bzr2650 - Linaro GCC 2014.03) ) #817 SMP PREEMPT Sat Sep 19 15:32:00 BST 2015" has a bug on
+    gatttool and the BLE connection will encounter a failure.
+  * WiFi on Raspberry Pi only tested with WiPi WiFi Dongle (Realink RT5370 chipset) and official Pi USB WiFi Dongle (Broadcom BCM43143 chipset).
```
Comment (pierantoniomerlino) [inline kura/distrib/RELEASE_NOTES.txt]: I don't think that this is already true...

File: kura/distrib/RELEASE_NOTES.txt [hunk 2]
```diff
@@ -1,3 +1,275 @@
...
+    * #1978: Modbus TCP incorrectly reads frame length
+    * #1940: Solved some possible NPEs in web2 bundle.
+    * #820:  Upgrade to GWT 2.8.2
+    
+    
+Know Issues :
+  * The implementation of the CryptoService performs encryption using a
+    password that is hardcoded and published.
+  * Modem: Ublox Lisa U201 may not be able to establish PPP connection when CHAP/PAP authentication is required.
+  * BLE only tested on the Raspberry Pi B with a Broadcom BCM20702 USB dongle and the TI SensorTag. The kernel version was
```
Comment (pierantoniomerlino) [inline kura/distrib/RELEASE_NOTES.txt]: This can be removed.

Comment (pierantoniomerlino) [reply kura/distrib/RELEASE_NOTES.txt]: "BLE also tested on..."

File: kura/distrib/RELEASE_NOTES.txt [hunk 3]
```diff
@@ -1,3 +1,277 @@
...
+      with remote providers that support the new Eclipse IoT WG Namespace. E.g. this provider supports the integration with Eclipse Hono through 
+      the MQTT Adapter 
+    * Kura Docker container: New Docker Container that wraps in a Centos 7 environment a Kura No Network installation. 
+    * Upload of Asset channels from CSV file: In the Asset view, the user can now download and upload channel descriptions as CSV files.
+
+  * Enhancements
+  	* Non-blocking assets configuration updates
+  	* Wires Graph Blinking rate limit 
+  	* Added fix validity, latitude and longitude hemisphere fields to GPS NMEA Position 
+
```
Comment (nicolatimeus) [inline kura/distrib/RELEASE_NOTES.txt:30]: We can add

* Reduced CPU usage by Kura Wires

as enhancement

**Pull request merged**